packages/stats/src/main/c/statsR.h

```diff
@@ -41,7 +41,11 @@ SEXP nextn(SEXP n, SEXP factors);
 
 SEXP cfilter(SEXP sx, SEXP sfilter, SEXP ssides, SEXP scircular);
 SEXP rfilter(SEXP x, SEXP filter, SEXP out);
+
+#ifndef R_STATS_H
```

Comment:
to avoid build conflict: in stats.h there is lowess, and arima.c include both stats.h and statsR.h.